src/store/modules/util/actions.ts

```diff
@@ -105,6 +105,27 @@ const actions: ActionTree<UtilState, RootState> = {
     }
     return state.facilities;
   },
+  async fetchProductStores({ state, commit }) {
+    let productStores = [];
+    
+    const payload = {
+      "viewSize": 50,
+      "entityName": "ProductStore",
```

Comment:
Pass fieldList as we only need specific fields in response like productStore name, id.